input/pagecontent/reference.md

```diff
@@ -2516,7 +2516,7 @@ As [advised in FHIR](https://hl7.org/fhir/R5/references.html#canonical), the URL
 
 The [CodeableReference](https://hl7.org/fhir/R5/references.html#codeablereference) datatype was introduced as part of FHIR R5 release sequence. This type allows for a concept, a reference, or both. FSH supports applying bindings directly to CodeableReferences and directly constraining types on CodeableReferences. When applying a type rule (e.g., `code only CodeableReference(Medication)`), the element's reference property is constrained to the given type, but its concept property remains available for use (unless otherwise constrained by another rule).
 
-To assign values to a CodeableReference, set the CodeableReference's concept and reference properties directly.
+To assign values to a CodeableReference, authors MAY assign a code or reference directly to the CodeableReference element. Assigning a code to a CodeableReference element assigns the code to the element's concept property. Assigning a reference to a CodeableReference element assigns the reference to the element's reference property. Assigning a code does not affect the CodeableReference's reference property, nor does assigning a reference affect the CodeableReference's concept property. If preferred, authors MAY assign a CodeableConcept's concept and reference properties directly.
```

Comment:
> ...authors MAY assign a code or reference directly to the CodeableReference element

This is genuinely a question, but is it misleading/confusing to say that authors may assign a _code_ or reference directly? Should it say _concept_ or maybe _coding_ or _CodeableConcept_? Or is that getting too deep in the weeds? In the example below, it describes the two parts as concept and reference, which doesn't quite line up with this paragraph.

Reply:
Yeah, that's fair. I think we use the word `code` pretty loosely throughout the spec -- but it's probably best to be a little more specific here. I think that `coding` may be more accurate, but it sounds awkward to me when I make that replacement, so I may go with `concept`.

Reply:
After reviewing our use of the words `concept` and `coding` in the documentation, I changed my mind and decided it would be best to go with `Coding`. This is most consistent with how we refer to FSH codes that have a system.

Reply:
Fixed and rebased into https://github.com/HL7/fhir-shorthand/pull/177/commits/90ba49916f08ac80d0e80d78cf16030c67cfab37.